codec_specs.md

```diff
@@ -1,21 +1,67 @@
 
-Each of these elements contain the same kind of data. And these data depend on the codec used.
+## Defining Matroska Codec Support
 
-Important Note:
+Support for a codec is defined in Matroska with the following values.
 
-The intention behind this list is `NOT` to list all existing audio and video codecs, but rather to list those codecs that are `currently supported` in Matroska and therefore need a well defined codec ID so that all developers supporting Matroska will use the same ID. If you feel we missed support for a very important codec, please tell us on our development mailing list (matroska-devel at lists.matroska.org).
+### Codec ID
 
+Each codec supported for storage in Matroska MUST have a unique Codec ID. Each Codec ID MUST be prefixed with a two character string according to the associated type of the codec as listed in the following table.
+
+Codec Type | Codec ID Prefix
+-----------|----------------
+Video      | V_
+Audio      | A_
+Subtitle   | S_
+Button     | B_
+
+Each Codec ID MUST then use a `Major Codec ID` and may optionally use a `Minor Codec ID` and `Micro Codec ID` with a forward-slash (`/`) delimiter between each component.
+
+For example a Codec ID MAY be constructed of the following components:
+
+Codec ID Prefix | Major Codec ID | Minor Codec ID | Micro Codec ID | Qualifier Codec ID | Codec ID
+A_              | AAC            | MPEG2          | LC             | SBR                | A_AAC/MPEG2/LC/SBR
+V_              | V_MPEG4        | ISO            | ASP            |                    | V_MPEG4/ISO/ASP
+V_              | MPEG1          |                |                |                    | V_MPEG1
+
+`Major Codec ID`, `Minor Codec ID`, and `Micro Codec ID` MUST only be composed of capital letters (A-Z) and numbers (0-9).
+
+### Codec Name
+
+Each encoding supported for storage in Matroska MUST have a Codec Name. The Codec Name provides a readable label for the encoding.
+
+### Description
+
+An optional description for the encoding.
+
+### Initialisation
+
+Each encoding supported for storage in Matroska MUST have a defined Initialisation. The Initialisation MUST describe the storage of data necessary to initialize the decoder, which MUST be stored within the `CodecPrivate Element`. When the Initialisation is updated within a track then that updated Initialisation data MUST be written into the `CodecState Element` of the first `Cluster` to require it. If the encoding does not require any form of Initialisation then `none` MUST be used to define the Initialisation and the `CodecPrivate Element` MUST be void.
+
+### Citation
+
+Documentation of the associated normative and informative referneces for the codec is RECOMMENDED.
+
+### Adoption Date
```

Comment:
I don't think it is easy to say the adoption date because it would be actually when the RFC is accepted, and it would be redundant with the RFC release date.
I suggest to remove this part.